odxtools/compumethods/createanycompumethod.py

```diff
@@ -52,9 +52,10 @@ def _parse_compu_scale_to_linear_compu_method(
     offset = computation_python_type(next(nums).text)
     factor_el = next(nums, None)
     factor = computation_python_type(factor_el.text if factor_el is not None else "0")
+    denominator = 1.0
     if coeffs.find("COMPU-DENOMINATOR/V") is not None:
-        kwargs["denominator"] = float(coeffs.findtext("COMPU-DENOMINATOR/V"))
-        assert kwargs["denominator"] > 0
+        denominator = float(coeffs.findtext("COMPU-DENOMINATOR/V"))
+        assert denominator > 0
```

Comment:
denominator may be negative

Reply:
and since python 3.8 has been required for a while, this can be done more efficiently like this:
```suggestion
    if (denom_str := coeffs.findtext("COMPU-DENOMINATOR/V")) is not None:
        denominator = float(denom_str)
```

(not a big deal, though)

further, it would be nice if you verified that the writing code for this was correct: https://github.com/mercedes-benz/odxtools/blob/main/odxtools/templates/macros/printDOP.xml.jinja2#L102